build.sbt

```
@@ -30,12 +30,14 @@ val stashScalaClient = project
   )
   .settings(unmanagedSourceDirectories.in(Compile) += {
     Dependencies.playJson(scalaVersion.value).head.revision match {
-      case Dependencies.playJson25 => baseDirectory.in(Compile).value / "src" / "main" / s"play_json_2.5"
+      case (Dependencies.playJson24 | Dependencies.playJson25) =>
+        baseDirectory.in(Compile).value / "src" / "main" / s"play_json_2.5-"
```

You can remove the `s`. The same in the next line
```suggestion
        baseDirectory.in(Compile).value / "src" / "main" / "play_json_2.5-"
```

---

👍

---

Changed there and the line bellow